### PR TITLE
Adjust build script to compile QTapp

### PR DESCRIPTION
**Pull request by jgandres**
- Update readme Md content about new QTapp and the way to construct
MEGAchat.

- Adjust path in .pri file

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,37 +1,93 @@
...
+## Automated build system ##
 
-## Build dependencies ##
-For Linux and MacOS, you need to install all dependencies using a package manager (the operating system's package manager for Linux, and MacPorts or Homebrew on MacOS). For cross-compiled targets (android, ios), and for Windows, an automated system is provided to download, build and install all dependencies under a specific prefix. See below for details.
+To build MEGAchat, you can do it with an automated script **build_with_webrtc.sh**.   
+Change directory to the root of the MEGAchat checkout  
+
+ - `cd ./third-party/mega/bindings/qt/`  
+
+ - `The sintax to invoke the script is:` **build_with_webrtc.sh <all | clean> [withExamples]**
```
Comment (sergiohs84) [inline Readme.md]:
```suggestion
 - ./build_with_webrtc.sh <all | clean> [withExamples]
```

File: Readme.md [hunk 2]
```diff
@@ -43,36 +99,8 @@ Version 2.0.x will **not** work, you need at least 2.1.x. You may need to build
...
-*IMPORTANT:* Do not run build-webrtc.sh in a shell with sourced env-ios.sh or env-android.sh  
-
-This system is generally an addon to the stock webrtc (actually chromium)
-build system, but it strips it down to download only a few hundred megabytes
-of source code and tools instead of 10-12GB. It also patches webrtc to fix
-several issues (use normal openssl instead of its own included boringssl
-lib, replace macos capturer that uses obsolete API and problematic threading
-model with modified iOS capturer, etc).
 
 ### Python version ###
```
Comment (sergiohs84) [inline Readme.md]:
```suggestion
### Python version (deprecated) ###
```

File: Readme.md [hunk 3]
```diff
@@ -1,37 +1,93 @@
...
+## Automated build system ##
+
+To build MEGAchat, you can do it with an automated script `build_with_webrtc.sh`.   
+Change directory to the root of the MEGAchat checkout  
+
+ - `cd ./third-party/mega/bindings/qt/` (This step is neccesary to set the right current path for the script)  
+
+ - The sintax to invoke the script is: `build_with_webrtc.sh <all | clean> [withExamples]`
+
+*NOTE: `[withExamples]` option will build the example QTapp to test MEGAchat
```
Comment (sergiohs84) [inline Readme.md]: I would add here:
```suggestion
*NOTE: `[withExamples]` option will build the example QTapp to test MEGAchat

Now you're ready. Open `<MEGAchat>/contrib/qt/MEGAchat.pro` in QtCreator and hit `Build` button.

You may need to change the "Build directory" in the project setting to `<MEGAchat>/build` if building complains about files not found.
```

File: Readme.md [hunk 4]
```diff
@@ -1,37 +1,97 @@
...
+
+## Get SDK code ##
+
+Change directory to the root of the MEGAchat checkout  
+
+ - `cd ./third-party/`  
+ - `git clone https://github.com/meganz/sdk.git`    
+ - `mv sdk mega`    
+ - `cd mega`    
+ - `./autogen`
```
Comment (polmr) [inline Readme.md]:
```suggestion
 - `./autogen.sh`    
```

File: Readme.md [hunk 5]
```diff
@@ -1,37 +1,97 @@
...
+
+*NOTE the `--recursive` switch - the repository contains git submodules that need to be checked out as well.
+
+## Get SDK code ##
+
+Change directory to the root of the MEGAchat checkout  
+
+ - `cd ./third-party/`  
+ - `git clone https://github.com/meganz/sdk.git`    
+ - `mv sdk mega`
```
Comment (polmr) [inline Readme.md]: Suggestion: do `git clone https://github.com/meganz/sdk.git mega`  and remove the `mv`

File: Readme.md [hunk 6]
```diff
@@ -87,90 +119,38 @@ and include this directory to be first in the system PATH in the shell where you
...
-   - non-iOS  
-  `cd src/out/Release|Debug`  
-  - iOS  
-  `cd src/out/Release-iphoneos|Debug-iphoneos`  
+Change directory to the root of the MEGAchat checkout  
 
-* Built executables  
+ - `mkdir ./build`    
+ - `cd ./build`    
+ - `qmake ./contrib/QtCreator/MEGAchat.pro -spec linux-g++ CONFIG+=qml_debug CONFIG+=force_debug_info CONFIG+=separate_debug_info && /usr/bin/make _all`
```
Comment (polmr) [inline Readme.md]: `./contrib` won't exist in the "cd'ed" build.   `..`?

File: Readme.md (hunk 5 above)
Comment (sergiohs84) [reply Readme.md]: Done

File: Readme.md (hunk 4 above)
Comment (sergiohs84) [reply Readme.md]: Done

File: Readme.md (hunk 6 above)
Comment (sergiohs84) [reply Readme.md]: You're right. Updating!

Comment (sergiohs84) [reply Readme.md]: You're right. Updating

**Pull request merged**